### PR TITLE
Add invitation char to config

### PR DESCRIPTION
**Pull request by ldmi3i**

### CODE REVIEW

File: tests/CliTestRunner.cpp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 
 #include <catch2/catch_test_macros.hpp>
+#include <stdio.h>
```
Comment (funbiscuit) [inline tests/CliTestRunner.cpp:3]: Not used

File: tests/EmbeddedCliTest.cpp [hunk 2]
```diff
@@ -2,6 +2,7 @@
 #include "CliMock.h"
 #include "CliTestRunner.h"
 
+#include <cstring>
```
Comment (funbiscuit) [inline tests/EmbeddedCliTest.cpp:5]: Not used

File: tests/CliTestRunner.cpp (hunk 1 above)
Comment (ldmi3i) [reply tests/CliTestRunner.cpp]: Yep, removed

File: tests/EmbeddedCliTest.cpp (hunk 2 above)
Comment (ldmi3i) [reply tests/EmbeddedCliTest.cpp]: This gone too)

**Pull request merged**